Focus/IxStore.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  IxStore.swift
+//  swiftz
+//
+//  Created by Alexander Ronald Altman on 6/12/14.
+//  Copyright (c) 2015 TypeLift. All rights reserved.
+//
+
+
+// N.B.:  In the indexed store comonad transformer, set, put, and peek are all distinct,
+// as are puts and peeks.  The lack of distinction here is due to the lack of transformer
+// nature; as soon as we get transformers, that will change.
+public struct IxStore<O, I, A> {
+	let pos : O
+
+	let set : I -> A
```

Comment:
This actually needs to stay as `set` (as long as Swift allows it), because conceptually it's "assigning an index".  You can take the naming convention up with @ekmett if you want.  :)